pyproject.toml

```diff
@@ -70,7 +70,7 @@ extend-ignore = [
 ]
 ignore = ["E501", "D107"]
 extend-exclude = ["__pycache__", "*.egg_info"]
-per-file-ignores = {"tests/*" = ["D100","D101","D102","D103","D104"]}
+per-file-ignores = {"tests/*" = ["D100","D101","D102","D103","D104","D205","D212","D415"]}
```

Comment:
What are these?

Reply:
Are more relevant example would be our charm template :smile: : https://github.com/canonical/platform-engineering-charm-template/blob/f871f59f45511bfde8ed9eef5efbdf6fea173206/pyproject.toml#L101